app/views/user_mailer/register_account.text.erb

```diff
@@ -0,0 +1,7 @@
+Welcome to TeamUp, <%= @user.username %>!
+
+In order to activate your account and all the other schtuff we provide, we need you to click the link below to verify your account.
+
+If you would rather not TeamUp, feel free to ignore this!
+
+<%= confirm_account_user_url(@user.confirmation_action) %>
```

Comment:
We can investigate the potential creation of a template for handling text vs. html, but according to the Rails documentation and Google, without an additional gem or third-party piece of software, it cannot be done. Opened #66 to address this.